app/models/raw_input_changes.rb

```diff
@@ -41,13 +41,22 @@ def matching_organization
   end
 
   def matching_website
-    @matching_website ||= Website.find_by(content: url)
+    @matching_website ||= find_website(url) || find_website(resolved_url)
+  end
+
+  def find_website(content)
+    Website.find_by(content: content)
   end
 
   def url
     @attrs.fetch(:website, {})[:content]
   end
 
+  def resolved_url
+    @resolved_url ||= WebsiteResolver.resolve url
+    @attrs[:website][:content] = @resolved_url if @resolved_url
```

Comment:
I've been thinking about this and I feel like we are "losing data," despite keeping the raw input. This mutation could be pointing out a minor flaw in our approach.

If we saved both websites (the `301` and the `200`) we would make fewer network calls when the `301` comes around again. This is the first opportunity we lose out on by not saving both websites to the org.

The second thing we lose out on is harder for me to pin down because it's speculative. In a world where we save both websites, I can imagine the `website` table having a field for `status`, which could have values like `301` and `200`. 

Why save the status? First of all by saving the status we know which website to export (the one that's `200`). Second, saving the status more accurately records our app's progress as it gathers data. Third, in the future, we'll probably be trying to figure out if an org is closed. Going from a `200` to a `404` could be a possible signal. And fourth, for any future crawlers, this status could be super helpful. 

These are just some thoughts I had. Perhaps it complicates things more than needed. What do you think?